src/utils/ts/sendClientError.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+export const sendClientError = (error: unknown) => {
+  const url = window.location.href;
+
+  const serializedError = JSON.stringify(
+    error,
+    Object.getOwnPropertyNames(error),
+  );
+
+  const deserializedError = Object.assign(
+    new Error(),
+    JSON.parse(serializedError),
+  );
+
+  axios.post('https://api-slack.internal.bcsdlab.com/api/error-notice/frontend', {
```

Comment:
비동기 함수로 작성하는게 추후 확장에 용이할 것 같아요~!
```suggestion
export const sendClientError = async (error: unknown) => {
  const url = window.location.href;

  const serializedError = JSON.stringify(
    error,
    Object.getOwnPropertyNames(error),
  );

  const deserializedError = Object.assign(
    new Error(),
    JSON.parse(serializedError),
  );

   return axios.post('https://api-slack.internal.bcsdlab.com/api/error-notice/frontend', {
```

Reply:
구두)
https://bcsdlab.slack.com/archives/C06SND3N6F7/p1712595790283049?thread_ts=1712593597.406839&cid=C06SND3N6F7